backbone/src/main/java/org/researchstack/backbone/ui/step/layout/ConsentDocumentStepLayout.java

```diff
@@ -78,6 +78,11 @@ public void setCallbacks(StepCallbacks callbacks) {
         this.callbacks = callbacks;
     }
 
+    @Override
+    public void setCancelEditMode(boolean isCancelEdit) {
+
```

Comment:
Add the same comment here: `//no-op: xxxx` (aka: why is this empty?)

(and in all the other empty overrides down below)